packages/admin-ui/ui/src/constants/injection-zones.ts

```diff
@@ -0,0 +1,52 @@
+export const injectionZones = [
```

Comment:
thought: Should we create a dict/map/enum, we can use it in the UI to not work with raw strings? Think raw strings are error-prone

Reply:
Cool, fine to keep as is